docs/platforms/azure.md

```diff
@@ -45,7 +45,7 @@ Once logged in you need to install node.js and Node-RED
 
        curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -
```

Comment:
really this should be 8.x now... - and that does include npm by default... so should not need the npm line

Reply:
Ok .